torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -1054,5 +1170,34 @@ TORCH_API bool aliasAnalysisHasSpecialCaseFor(Symbol symbol) {
   return handled.count(symbol) || purposefully_not_handled.count(symbol);
 }
 
+// Register `v` as a wildcard value.
+void AliasDb::setWildcard(const Value* v) {
+  wildcards_.insert(v);
+}
+
+void AliasDb::rebuildWriteCache() const {
```

Comment:
Why is this function `const` if it has no returns/arguments?

Reply:
to signify that it does not mutate externally visible state of the object (e.g. is usable from a `const AliasDb`)